src/wikibots/youtube.py

```
@@ -83,38 +84,37 @@ def treat_page(self) -> None:
 
         :param str youtube_id: The ID of the YouTube video to fetch
-        :return: A dictionary containing video details, or None if the video is not found
-        :rtype: YouTubeVideo | None
+        :return: None
         """
         try:
             start = perf_counter()
             video = self.youtube.videos().list(part="snippet", id=youtube_id).execute()
             info(f"Retrieved video data in {(perf_counter() - start) * 1000:.0f} ms")
-        except googleapiclient.errors.HttpError as e:
+        except HttpError as e:
```

Ruff (0.8.2)

Undefined name `HttpError`

(F821)